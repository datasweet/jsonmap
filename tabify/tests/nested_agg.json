{
  "took": 0,
  "timed_out": false,
  "_shards": {
    "total": 5,
    "successful": 5,
    "skipped": 0,
    "failed": 0
  },
  "hits": {
    "total": 2,
    "max_score": 0,
    "hits": []
  },
  "aggregations": {
    "byType": {
      "doc_count_error_upper_bound": 0,
      "sum_other_doc_count": 0,
      "buckets": [
        {
          "key": "Final",
          "doc_count": 1,
          "affirmations": {
            "doc_count": 160,
            "byPivot": {
              "doc_count_error_upper_bound": 0,
              "sum_other_doc_count": 0,
              "buckets": [
                {
                  "key": "1",
                  "doc_count": 32,
                  "sumWeight": {
                    "value": 0
                  }
                },
                {
                  "key": "2",
                  "doc_count": 16,
                  "sumWeight": {
                    "value": 12.900000154972076
                  }
                },
                {
                  "key": "3",
                  "doc_count": 48,
                  "sumWeight": {
                    "value": 42.4000004529953
                  }
                },
                {
                  "key": "4",
                  "doc_count": 40,
                  "sumWeight": {
                    "value": 32.400000393390656
                  }
                },
                {
                  "key": "5",
                  "doc_count": 20,
                  "sumWeight": {
                    "value": 18.000000178813934
                  }
                },
                {
                  "key": "6",
                  "doc_count": 4,
                  "sumWeight": {
                    "value": 3.900000035762787
                  }
                }
              ]
            }
          }
        },
        {
          "key": "Initial",
          "doc_count": 1,
          "affirmations": {
            "doc_count": 160,
            "byPivot": {
              "doc_count_error_upper_bound": 0,
              "sum_other_doc_count": 0,
              "buckets": [
                {
                  "key": "1",
                  "doc_count": 32,
                  "sumWeight": {
                    "value": 0
                  }
                },
                {
                  "key": "2",
                  "doc_count": 16,
                  "sumWeight": {
                    "value": 12.20000010728836
                  }
                },
                {
                  "key": "3",
                  "doc_count": 48,
                  "sumWeight": {
                    "value": 39.10000038146973
                  }
                },
                {
                  "key": "4",
                  "doc_count": 40,
                  "sumWeight": {
                    "value": 36.500000298023224
                  }
                },
                {
                  "key": "5",
                  "doc_count": 20,
                  "sumWeight": {
                    "value": 17.200000166893005
                  }
                },
                {
                  "key": "6",
                  "doc_count": 4,
                  "sumWeight": {
                    "value": 2.600000023841858
                  }
                }
              ]
            }
          }
        }
      ]
    }
  }
}
