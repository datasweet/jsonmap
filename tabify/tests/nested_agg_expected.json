[
  {
    "byType": "Final",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "1",
    "byType#affirmations#byPivot#doc_count": 32,
    "byType#affirmations#byPivot#sumWeight": 0
  },
  {
    "byType": "Final",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "2",
    "byType#affirmations#byPivot#doc_count": 16,
    "byType#affirmations#byPivot#sumWeight": 12.900000154972076
  },
  {
    "byType": "Final",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "3",
    "byType#affirmations#byPivot#doc_count": 48,
    "byType#affirmations#byPivot#sumWeight": 42.4000004529953
  },
  {
    "byType": "Final",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "4",
    "byType#affirmations#byPivot#doc_count": 40,
    "byType#affirmations#byPivot#sumWeight": 32.400000393390656
  },
  {
    "byType": "Final",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "5",
    "byType#affirmations#byPivot#doc_count": 20,
    "byType#affirmations#byPivot#sumWeight": 18.000000178813934
  },
  {
    "byType": "Final",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "6",
    "byType#affirmations#byPivot#doc_count": 4,
    "byType#affirmations#byPivot#sumWeight": 3.900000035762787
  },
  {
    "byType": "Initial",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "1",
    "byType#affirmations#byPivot#doc_count": 32,
    "byType#affirmations#byPivot#sumWeight": 0
  },
  {
    "byType": "Initial",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "2",
    "byType#affirmations#byPivot#doc_count": 16,
    "byType#affirmations#byPivot#sumWeight": 12.20000010728836
  },
  {
    "byType": "Initial",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "3",
    "byType#affirmations#byPivot#doc_count": 48,
    "byType#affirmations#byPivot#sumWeight": 39.10000038146973
  },
  {
    "byType": "Initial",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "4",
    "byType#affirmations#byPivot#doc_count": 40,
    "byType#affirmations#byPivot#sumWeight": 36.500000298023224
  },
  {
    "byType": "Initial",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "5",
    "byType#affirmations#byPivot#doc_count": 20,
    "byType#affirmations#byPivot#sumWeight": 17.200000166893005
  },
  {
    "byType": "Initial",
    "byType#doc_count": 1,
    "byType#affirmations#doc_count": 160,
    "byType#affirmations#byPivot": "6",
    "byType#affirmations#byPivot#doc_count": 4,
    "byType#affirmations#byPivot#sumWeight": 2.600000023841858
  }
]
